include/swift/SIL/SILModule.h

```diff
@@ -720,6 +720,9 @@ inline llvm::raw_ostream &operator<<(llvm::raw_ostream &OS, const SILModule &M){
   return OS;
 }
 
+void simple_display(llvm::raw_ostream &out, const SILModule *M);
+SourceLoc extractNearestSourceLoc(const SILModule *SM);
```

Comment:
These need comments. Tell me why I would want to use this and what for?

Reply:
@gottesmm Sure thing! Though they shouldn't be called directly, they should only really be called by the evaluator's internals.